### PR TITLE
Update translation verification

### PR DESCRIPTION
**Pull request by cea2aj**
Update translation verification so that it also checks that the messages.pot file in the repo is up to date

Also update translation test to run on every branch

J=none
TEST=manual

Ran the new extract translations script and see the correct behavior. Update the circle ci script and check the exit codes to ensure that code 0 is returned when successful, and 1 is returned when the messages.pot is out of date or when translations are missing

### CODE REVIEW

File: .github/run_translation_verification.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+#!/bin/bash
+
+set -e
+
+# Verifiy that the messages.pot file in the repo is up to date
+# If there are any git diffs after translations are extracted, the checked-in file is out of date
+npm run extract-translations
+
+git diff --exit-code translations/messages.pot > /dev/null # send stdout to /dev/null to reduce clutter in the CI output
```
Comment (oshi97) [inline .github/run_translation_verification.sh:10]: I think we can use something like the $GITHUB_REF env var to always run the messages.pot verification, but only run the [lang].po check for master and release branches

Comment (oshi97) [reply .github/run_translation_verification.sh:10]: Ah actually GITHUB_BASE_REF might be better

Comment (cea2aj) [reply .github/run_translation_verification.sh:10]: In the run_browserstack_acceptance.sh script we use $GITHUB_BRANCH. Does that work?

Comment (oshi97) [reply .github/run_translation_verification.sh:10]: I thiink that might be the branch being merged into the base branch? Not sure though

Comment (cea2aj) [reply .github/run_translation_verification.sh:9]: updated!

Comment (oshi97) [reply .github/run_translation_verification.sh:9]: oh wait I did not realize we were already using it for filtering lol, think I was looking at the version on master

File: .github/run_translation_verification.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (oshi97) [inline .github/run_translation_verification.sh:1]: besides this extra line pr looks great!

**Review by oshi97 — APPROVED** [review]

**Pull request merged**